mmv1/third_party/terraform/website/docs/d/backup_dr_backup_vault.html.markdown

```diff
@@ -0,0 +1,31 @@
+---
+subcategory: "Backup and DR BackupVault"
+description: |-
+  Get information about a Backupdr BackupVault.
+---
+
+# google_backup_dr_backup_vault
+
+A Backup and DRBackupVault.
+
+## Example Usage
+
+```hcl
+data "google_backup_dr_backup_vault" "my-backup-vault" {
+  location =  "us-central1"
+  backup_vault_id="bv-1"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `location` - (Required) The location in which the Backup Vault resource belongs.
+* `backup_vault_id` - (Required) The id of Backup Vault resource.
+
```

Comment:
Can you also add `project` field here? Something like https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/d/cloud_run_v2_job.html.markdown?plain=1#L30-L33